template-for-repository/proofs/Makefile.common

```diff
@@ -402,31 +425,77 @@ arpa: $(ARPA_BLDDIR)
 # Targets to run the analysis commands
 
 $(LOGDIR)/result.txt: $(HARNESS_GOTO).goto
-	$(call DO_CBMC,$(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace,$<,$@)
+	$(LITANI) add-job \
+	  --command \
+	    '$(CBMC) $(CBMC_VERBOSITY) $(CBMCFLAGS) $(CHECKFLAGS) --trace $<' \
+	  --inputs $^ \
+	  --outputs $@ \
+	  --pipeline-name "$(PROOF_UID)" \
+	  --ci-stage test \
+	  --stdout-file $@ \
```

Comment:
Again, in all of these litani invocations, don't we want to collect and interleave all stdout stderr output?

Reply:
done